bot.py

```diff
@@ -83,13 +85,21 @@ async def help_command(ctx):
                     }
                 )
         await ctx.send(embed=create_embed(embed_data))
-        # pylint: disable=broad-except
-    except Exception as e_values:
-        print(e_values)
+
+    # pylint: disable=broad-except
+    except Exception:
+        err_embed = discord.Embed(
+            title="Error",
+            description="Oops! Something went wrong.\n\nPlease try again!\
+              \n\n Please type  `rank --help`  to see how to use",
```

Comment:
description can be shorted just like https://github.com/dhmoon91/discord/pull/27/files#r668373752